scripts/upload_demo_data.py

```diff
@@ -239,5 +240,7 @@ def asset_to_project(project_id, asset_id, asset_type):
     except Exception as e:
         print(e)
 
+programatically_populate_datasets()
```

Comment:
how do we handle the case where the user doesn't have data annotation?

Reply:
This script is independent of the data-annotation stack, it doesn't make any calls to the data-annotation API.

Reply:
It's dependent on experiments directory though, so if the BIOMD sim-output.json files change it will break.